diff --git a/imx9/evk_95/AndroidUboot.sh b/imx9/evk_95/AndroidUboot.sh
index da8d64d5..1c811d0f 100644
--- a/imx9/evk_95/AndroidUboot.sh
+++ b/imx9/evk_95/AndroidUboot.sh
@@ -66,11 +66,8 @@ fi
 	echo Building imx-oei ...
 	make -C ${BOARD_OEI_PATH} really-clean
 	make -C ${BOARD_OEI_PATH} OEI_CROSS_COMPILE="${SM_OEI_CROSS_COMPILE}" board=mx95lp5 r=b0 oei=tcm d=1 all 1>/dev/null || exit 1
-if echo "$2" | grep -q "15x15" ; then
 	make -C ${BOARD_OEI_PATH} OEI_CROSS_COMPILE="${SM_OEI_CROSS_COMPILE}" board=mx95lp4x-15 r=b0 oei=ddr d=1 all 1>/dev/null || exit 1
-else
 	make -C ${BOARD_OEI_PATH} OEI_CROSS_COMPILE="${SM_OEI_CROSS_COMPILE}" board=mx95lp5 r=b0 oei=ddr d=1 all 1>/dev/null || exit 1
-fi
 
 if [ "${PRODUCT_IMX_CAR_M7}" = "true" ]; then
 	echo Building Cortex-M7 Car firmware ...
