diff --git a/common/tools/uuu_imx_android_flash.sh b/common/tools/uuu_imx_android_flash.sh
index c70d1d26..1ef693f1 100755
--- a/common/tools/uuu_imx_android_flash.sh
+++ b/common/tools/uuu_imx_android_flash.sh
@@ -457,11 +457,11 @@ randome_part=
 # images when flash another set of images can not be detect early with this scenario.
 imx8qxp_uboot_feature=(mek-uuu secure-unlock c0 mek-c0-uuu)
 imx8qm_uboot_feature=(mek-uuu secure-unlock md)
-imx95_uboot_feature=(evk-uuu secure-unlock verdin verdin-uuu)
+imx95_uboot_feature=(evk-uuu secure-unlock verdin verdin-uuu 15x15-frdm)
 
 imx8qxp_dtb_feature=(sof)
 imx8qm_dtb_feature=(md sof revd md-revd sof-revd)
-imx95_dtb_feature=(ap1302 lvds0 lvds0-ap1302 mipi-lvds1 mipi-lvds1-ap1302 verdin verdin-ap1302 verdin-adv7535 verdin-adv7535-ap1302)
+imx95_dtb_feature=(ap1302 lvds0 lvds0-ap1302 mipi-lvds1 mipi-lvds1-ap1302 verdin verdin-ap1302 verdin-adv7535 verdin-adv7535-ap1302 15x15-frdm)
 
 tmp_files_before_uuu=()
 tmp_files_in_uuu=()
